src/RelayBuilder.php

```diff
@@ -95,13 +96,16 @@ protected function getArray($queue)
             return $queue;
         }
 
-        $getArrayCopy = $queue instanceof GetArrayCopyInterface
-            || $queue instanceof ArrayObject;
-
-        if ($getArrayCopy) {
+        if ($queue instanceof ArrayObject
+            || $queue instanceof GetArrayCopyInterface
```

Comment:
Man, I don't mean to be a nitpicker, but can you restore those lines to exactly as they were? If nothing else it makes the diff cleaner. Thanks for your patience with my OCD.